test/stdtypes/concurrency_test.go

```diff
@@ -0,0 +1,51 @@
+package stdtypes
+
+import (
+	"io/ioutil"
+	"sync"
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+)
+
+func TestConcurrentTextMarshal(t *testing.T) {
```

Comment:
@awalterschulze can you give me a hint on how to run this one test with the -race flag? It looks like all tests are run under one `go test ./...` invocation.  Should I add it to the individual `test/stdtypes/Makefile`?